extensions/disable-cnf_chroot.sh

```diff
@@ -0,0 +1,15 @@
+function pre_install_distribution_specific__001_disable_cnf_apt_hook(){
```

Comment:
Too low; try to always allow for the future extensions to run earlier. Say, `250_` should be early enough for this.

Reply:
removed numeral, should now run with framework fedault priority